index.html

```diff
@@ -2241,6 +2246,27 @@ <h1>Core Representations</h1>
 through the content of the <a>DID document</a> alone.
     </p>
 
+    <p>
+Consumers MUST add all properties that do not have explicit processing rules
+for the representation being consumed to the abstract data model using only the
+representation's generic type processing rules. Producers MUST serialize all
```

Comment:
The Abstract Data Model is too Abstract. We need to make this more concrete before enforcing the MUST.

Reply:
I think we did that with #455